app/src/main/java/org/wikipedia/views/ImageTitleDescriptionView.kt

```diff
@@ -0,0 +1,88 @@
+package org.wikipedia.views
+
+
+import android.content.Context
+import android.content.res.ColorStateList
+import android.graphics.drawable.Drawable
+import android.util.AttributeSet
+import android.util.TypedValue
+import android.view.Gravity.CENTER_HORIZONTAL
+import android.view.View
+import android.view.ViewGroup
+import android.widget.FrameLayout
+import android.widget.LinearLayout
+import androidx.core.content.ContextCompat
+import kotlinx.android.synthetic.main.view_image_title_description.view.*
+import org.wikipedia.R
+import org.wikipedia.util.DimenUtil
+
+internal class ImageTitleDescriptionView @JvmOverloads constructor(context: Context, attrs: AttributeSet? = null, defStyle: Int = 0) : LinearLayout(context, attrs, defStyle) {
+
+    init {
+        View.inflate(context, R.layout.view_image_title_description, this)
+        layoutParams = ViewGroup.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT)
+        circularProgressBar.setCurrentProgress(80.0)
+    }
+
+    fun setTitle(titleText: String) {
+        title.text = titleText
+    }
+
+    fun setTitleTextSize(textSize: Float) {
+        title.setTextSize(TypedValue.COMPLEX_UNIT_SP, textSize)
+    }
+
+    fun setCaseForTitle(isAllCaps: Boolean) {
+        title.isAllCaps = isAllCaps
+    }
+
+    fun setDescription(descriptionText: String) {
+        description.text = descriptionText
+    }
+
+    fun setImageDrawable(imageDrawable: Drawable) {
+        image.setImageDrawable(imageDrawable)
+    }
+
+    fun setImageTint(color: Int) {
+        image.imageTintList = ColorStateList.valueOf(ContextCompat.getColor(context, color))
+    }
+
+    fun setImageParams(width: Int, height: Int) {
+        val params = LayoutParams(width, height)
+        image.layoutParams = params
+        image.requestLayout()
+    }
+
+    fun setImageBackgroundParams(width: Int, height: Int) {
+        val params = FrameLayout.LayoutParams(width, height)
+        imageBackground.layoutParams = params
+        imageBackground.requestLayout()
+    }
+
+    fun setImageBackgroundTint(color: Int) {
+        imageBackground.backgroundTintList = ColorStateList.valueOf(ContextCompat.getColor(context, color))
+    }
+
+    fun setImageBackground(drawable: Drawable?) {
+        imageBackground.background = drawable
+
+    }
```

Comment:
Can we have the default value of `imageBackground.background` set as `null`. So that we will not need to set a `null` value explicitly.

Reply:
Also, there's an unused empty line in `69`.